docs/feature_wpm.md

```diff
@@ -0,0 +1,25 @@
+# Word Per Minute (WPM) Calculcation
```

Comment:
A link to this needs to be added to `_summary.md`, as well.

Reply:
Thank you @drashna .  I very much appreciate your patient guidance on my first contribution outside of keymaps!  I've added a commit to address this.